src/pages/JobOwner/MyJobs/ClientJobs.tsx

```diff
@@ -15,65 +17,69 @@ import { useTranslation } from "react-i18next";
 import { useGetOwnJobPostsQuery } from "services/jobPosts/setJobPostsAPI";
 import { CREATE_JOB_POST } from "utils/consts/routeConsts";
 import { POSTS_PER_PAGE as PPG } from "utils/consts/jobListConsts";
-import defineContractStatus from "utils/functions/defineContractStatus";
+import { useDispatch, useSelector } from "react-redux";
+import { RootState } from "store/store";
+import { toggleIsDraft } from "store/slices/isDraftSlice";
+
+const { Title } = Typography;
 
 function ClientJobs() {
+  const { isDraft } = useSelector((state: RootState) => state.isDraft);
+  const dispatch = useDispatch();
+
   const { email } = useAuth();
   const { t } = useTranslation();
-  const [currentPage, setCurrentPage] = useState(1);
-  const [postsPerPage, setPostsPerPage] = useState(PPG);
-
-  const { data: jobPosts, isLoading, isSuccess } = useGetOwnJobPostsQuery();
+  const [currentPage, setCurrentPage] = useState<number>(1);
+  const [postsPerPage, setPostsPerPage] = useState<number>(PPG);
 
+  const {
+    data: jobPosts,
+    isLoading,
+    isSuccess,
+  } = useGetOwnJobPostsQuery(isDraft);
   const lastPostIndex = currentPage * postsPerPage;
   const firstPostIndex = lastPostIndex - postsPerPage;
 
   return (
     <PageContainer>
       <Header>
-        <TitleText>
-          {t("MyJobs.subTitle")}
+        <div>
+          <Title level={3}>{t("MyJobs.subTitle")}</Title>
           <NameText>{email}</NameText>
-        </TitleText>
+        </div>
         <LinkButton link={CREATE_JOB_POST}>{t("MyJobs.postJob")}</LinkButton>
       </Header>
       <Section>
         <Card>
           <TitleText font_sz="1.5em" pd="5px" pd_bottom="5%">
             {t("MyJobs.allPostings")}
+            <Filter>
+              {t("MyJobs.draft?")}
+              <Switch
+                checked={isDraft}
+                onChange={() => dispatch(toggleIsDraft())}
+              />
+            </Filter>
           </TitleText>
-          {isLoading ? <Skeleton active paragraph={{ rows: 4 }} /> : ""}
+          {isLoading ? (
+            <DivContainer>
+              <Skeleton active paragraph={{ rows: 4 }} />
+            </DivContainer>
+          ) : (
+            ""
```

Comment:
try I think it should work with binary operator
```
isLoading ?? (
            <DivContainer>
              <Skeleton active paragraph={{ rows: 4 }} />
            </DivContainer>
          ) 
```

Reply:
The AND operation works better with isLoading, because it will never be null since it is a boolean value

```JSX
{isLoading && (
            <DivContainer>
              <Skeleton active paragraph={{ rows: 4 }} />
            </DivContainer>
          )}
```